### PR TITLE
Add login

### PR DESCRIPTION
**Pull request by amelieKontze**

### CODE REVIEW

File: backend/src/main/java/de/neuefische/backend/controller/KeeperController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+@RestController
+@RequestMapping("/keeper")
+@RequiredArgsConstructor
+public class KeeperController {
+
+    private final ServiceKeeper serviceKeeper;
+
+    @GetMapping("/user")
+    public String getUserFromEverywhere() {
+        System.out.println(SecurityContextHolder.getContext());
```
Comment (GitChrisCode) [inline backend/src/main/java/de/neuefische/backend/controller/KeeperController.java:20]: Is this really necessary?

File: backend/src/main/java/de/neuefische/backend/service/ServiceKeeper.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    private final RepoKeeper repoKeeper;
+
+    @Override
+    public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+        Keeper keeper = repoKeeper.findKeeperByUsername(username)
+                .orElseThrow(() -> new UsernameNotFoundException("User " + username + " not found"));
+        return new User(keeper.getUsername(), keeper.getPassword(), List.of());
+    }
+}
```
Comment (GitChrisCode) [inline backend/src/main/java/de/neuefische/backend/service/ServiceKeeper.java:26]: Why is this code not covered by a test?

**Review by GitChrisCode — COMMENTED** [review]
Just a few things to look at ...

File: backend/src/main/java/de/neuefische/backend/controller/KeeperController.java (hunk 1 above)
Comment (amelieKontze) [reply backend/src/main/java/de/neuefische/backend/controller/KeeperController.java:20]: Yep, bc we want to display the current user on the homepage. We´re not using it atm but will soon :)

**Review by GitChrisCode — APPROVED** [review]
ok ... have a nice day :-)

**Pull request merged**